src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -567,9 +572,9 @@ internal String ToString(TraceFormat traceFormat)
                         String typeName = "<UnknownType>";
                         if (pi[j].ParameterType != null)
                             typeName = pi[j].ParameterType.Name;
-                        sb.Append(typeName + " " + pi[j].Name);             
+                        sb.Append(typeName).Append(' ').Append(pi[j].Name);             
```

Comment:
I don't think I ever realized that Append returned StringBuilder. This is the first time I've ever seen it used in a fluent API way. For consistency with other places can you please put the Append calls on their own lines?